### PR TITLE
Updated to Angular 11

### PR DESCRIPTION
**Pull request by jfbenckhuijsen**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": "dc2c4426-745f-406d-82f0-fdf9c84be24f"
```
Comment (maxime1992) [inline angular.json]: I'd rather prefer not to have this

Comment (jfbenckhuijsen) [reply angular.json]: 👍🏻

**Review by maxime1992 — CHANGES_REQUESTED** [review]
Tested locally, LGTM :+1: 

One last thing can you please run `yarn prettier:write` as there's a diff on that :+1: 

Will merge after :)

**Comment by jfbenckhuijsen** [comment]
Fixed code style for ya ;)

**Review by maxime1992 — APPROVED** [review]

**Comment by maxime1992** [comment]
I messed up the squash :disappointed: 

I wanted to add a breaking change for the major bump but failed... By pressing enter to go to next line and it just sent the form :facepalm: 

sjhbsjhzegjhzbsjkbhgszjbfjsbvgfjhksbvag :upside_down_face: 

Thanks for the upgrade and the time spent on this @jfbenckhuijsen! :clap:

**Pull request merged**